Makefile

```diff
@@ -0,0 +1,48 @@
+
+.PHONY: build test clean docker
+
+MICROSERVICES=build/release/device-coap
+.PHONY: $(MICROSERVICES)
+
+DOCKERS=docker_device_coap_c
```

Comment:
We are fixing the way we name the docker images to not have `docker_` or  `_c` in the name. Please change to just `device_coap`

Reply:
Addressed in 561f925